[0001].[2021.04.03.01.46.51.834174].SRV1.INF:Main.py.<module>.315:Main Program Started
[0002].[2021.04.03.01.46.51.834174].SRV1.ERR:Main.py.<module>.318:Artificial Error Message for testing purpose
[0003].[2021.04.03.01.46.51.835174].SRV1.INF:Main.py.<module>.319:Command Line iniFileName=default.ini
[0004].[2021.04.03.01.46.51.835174].SRV1.INF:Main.py.<module>.321:Command Line VerboseMode=None
[0005].[2021.04.03.01.46.51.836175].SRV1.INF:Main.py.<module>.324:AppSpoManager.DebugVerboseModeOn=True
[0006].[2021.04.03.01.46.51.836175].SRV1.INF:Main.py.<module>.325:AppSpoManager.DebugInfoMessages=True
[0007].[2021.04.03.01.46.51.837175].SRV1.INF:Main.py.<module>.326:AppSpoManager.DebugErrorMessages=True
[0008].[2021.04.03.01.46.51.837175].SRV1.INF:Main.py.<module>.327:AppSpoManager.DebugTargetConsole=False
[0009].[2021.04.03.01.46.51.838176].SRV1.INF:Main.py.<module>.328:AppSpoManager.DebugTargetFile=True
[0010].[2021.04.03.01.46.51.838176].SRV1.INF:Main.py.<module>.329:AppSpoManager.DebugTargetFilePath=Debugs\Debug1
[0011].[2021.04.03.01.46.51.838176].SRV1.INF:Main.py.<module>.330:AppSpoManager.DebugLinesPerFile=1000
[0012].[2021.04.03.01.46.51.839177].SRV1.INF:Main.py.<module>.331:AppSpoManager.AppRunFolder=c:\Apps\PythonTry1
[0013].[2021.04.03.01.46.51.839177].SRV1.INF:Main.py.<module>.354:ModuleFileNameOnly=Main.py
[0014].[2021.04.03.01.46.51.840177].SRV1.INF:Main.py.<module>.362:File=c:\Apps\PythonTry1\Main.py
[0015].[2021.04.03.01.46.51.840177].SRV1.INF:Main.py.<module>.363:FileDirName=c:\Apps\PythonTry1
[0016].[2021.04.03.01.46.51.841178].SRV1.INF:Main.py.<module>.364:FileAbsPath=c:\Apps\PythonTry1\Main.py
[0017].[2021.04.03.01.46.51.841178].SRV1.INF:Main.py.<module>.367:ModuleName1=__main__
[0018].[2021.04.03.01.46.51.842179].SRV1.INF:Main.py.<module>.368:ModuleName2=Main
[0019].[2021.04.03.01.46.51.842179].SRV1.INF:Main.py.<module>.377:Main Program Started
[0020].[2021.04.03.01.46.51.843179].SRV1.INF:Main.py.<module>.378:Python Executable Path=C:\Program Files\Python39\python.exe
[0021].[2021.04.03.01.46.51.843179].SRV1.INF:Main.py.<module>.379:Python Interpreter Version=3.9.2 (tags/v3.9.2:1a79785, Feb 19 2021, 13:44:55) [MSC v.1928 64 bit (AMD64)]
[0022].[2021.04.03.01.46.51.844180].SRV1.INF:Main.py.<module>.382:Sys.Path=['c:\\Apps\\PythonTry1', 'C:\\Program Files\\Python39\\python39.zip', 'C:\\Program Files\\Python39\\DLLs', 'C:\\Program Files\\Python39\\lib', 'C:\\Program Files\\Python39', 'C:\\Users\\user2\\AppData\\Roaming\\Python\\Python39\\site-packages', 'C:\\Program Files\\Python39\\lib\\site-packages']
[0023].[2021.04.03.01.46.51.872198].SRV1.INF:Main.py.<module>.429:Showing MainWindow
